install/kubernetes/plugin-ca-certs.yaml

```diff
@@ -0,0 +1,49 @@
+################################
+# Istio CA with plugged key/certs. Run this after istio-auth.yaml
+################################
+# Service account CA
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-ca-service-account
+  namespace: istio-system
+---
+apiVersion: v1
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: istio-ca
+  namespace: istio-system
+  annotations:
+    sidecar.istio.io/inject: "false"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        istio: istio-ca
+    spec:
+      serviceAccountName: istio-ca-service-account
+      containers:
+      - name: istio-ca
+        image: gcr.io/istio-testing/istio-ca:d8928ed10bdbd180cbad39f9a4eaf8a70ddb37a2
```

Comment:
what's that version ? should be generated from istio.VERSION no ? or be :0.2.11 ?

Reply:
Sorry I should put {CA_HUB}/istio-ca:{CA_TAG} here.